packages/pwa-kit-extension-sdk/src/react/classes/ApplicationExtension.test.tsx

```diff
@@ -35,6 +34,7 @@ describe('ApplicationExtension', () => {
         })
     })
 
+    /*
```

Comment:
What are you plans here with this commented out test?

Reply:
I plan to write tests as part of this PR. So I'll update that commented-out tests.